lib/index.js

```diff
@@ -25,9 +28,11 @@ function create_oauth2_client () {
  *  @returns {String} url - the url where people visit to authenticate
  */
 function generate_google_oauth2_url () {
+  var access_type = (OPTIONS.access_type) ? OPTIONS.access_type : 'offline';
+  var approval_prompt = (OPTIONS.approval_prompt) ? OPTIONS.approval_prompt : 'force';
```

Comment:
`access_type` should be `ACCESS_TYPE`
and
`approval_prompt` >> `APPROVAL_PROMPT` just for consistency.
otherwise this code is good and I'm happy to _merge_ it! 👍